CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Viash 0.x.x (yyyy-MM-dd): TODO Add title
 
+## BREAKING CHANGES
+
+* `arguments`: Change default `multiple_sep` from `:` to `;` to avoid conflicts with paths like `s3://foo/bar` (PR #645).
+  As reference, a config mod to change the `multiple_sep` back to the previous default could be `-c '.functionality.argument_groups[true].arguments[.multiple == true].multiple_sep := ","'`.
```

Comment:
maybe be explicit with the also mentioning the case for arguments, in case a user doesn't think about it:

```suggestion
  As reference, a config mod to change the `multiple_sep` back to the previous default could be added to the `_viash.yaml`:
    config_mods: |
      .functionality.argument_groups[true].arguments[.multiple == true].multiple_sep := ","
      .functionality.arguments[.multiple == true].multiple_sep := ","
```

Reply:
I believe it should be

```suggestion
  The previous behaviour of using `multiple_sep: ":"` can be achieved by adding a config mod to the `_viash.yaml`:
    config_mods: |
      .functionality.argument_groups[true].arguments[.multiple == true].multiple_sep := ":"
```
As Viash 0.9 won't have `.functionality.arguments`.